### PR TITLE
feat: slack/notify-result

### PR DESCRIPTION
**Pull request by v1v**
A more straightforward implementation of https://github.com/elastic/apm-pipeline-library/tree/0b1a309a85517985492e5d79572b13d59648e50e/.github/actions/notify-build-status

image

### CODE REVIEW

File: slack/notify-result/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+<!--usage action="elastic/oblt-actions/slack/notify-result" version="env:VERSION"-->
+```yaml
+name: slack
+
+jobs:
+  slack:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: elastic/oblt-actions/slack/buld-result@v1
```
Comment (v1v) [inline slack/notify-result/README.md:22]:
```suggestion
      - uses: elastic/oblt-actions/slack/notify-result@v1
```

File: slack/notify-result/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# <!--name-->slack/notify-result<!--/name-->
+
```
Comment (reakaleek) [inline slack/notify-result/README.md:2]:
```suggestion
<!--description-->
```

needs an gh action-readme update

Comment (v1v) [reply slack/notify-result/README.md:2]: what do you mean? I used the pre-commit locally

Comment (reakaleek) [reply slack/notify-result/README.md:2]: or pre-commit, yeah 😅

Comment (reakaleek) [reply slack/notify-result/README.md:2]: Sorry, I wasn't clear enough.

The description is missing in the `README.md`.

I added the suggestion to add `<!--description->` tag.

But if you apply the suggestion, it probably still needs to be updated by pre-commit or gh action-readme.

**Comment by reakaleek** [comment]
Will the ref show `main` for the main branch?

**Comment by reakaleek** [comment]
IMO, the explicit status text would be beneficial for color-blind people, especially green/read weakness.

**Comment by v1v** [comment]
> IMO, the explicit status text would be beneficial for color-blind people, especially green/red color-blindness.

I see, how can we ensure the size of the message box smaller and also support the color being explicit?

**Comment by v1v** [comment]
> IMO, the explicit status text would be beneficial for color-blind people, especially green/red color-blindness.

let's discuss this in a follow-up

**Pull request merged**